### PR TITLE
Update Tumblr and SmugMug OSINT Templates with Improved Username Validation

### PR DESCRIPTION
**Pull request by MillerMedia**
### Template / PR Information

• Added templates to detect user profile information for Tumblr and SmugMug using OSINT techniques.

• The templates validate usernames and avoid false positives for cases where periods (.) are incorrectly interpreted as subdomain separators.

• References: N/A

### Template Validation

I've validated this template locally?
- [ x] YES
- [ ] NO

### Additional References:

- [Nuclei Template Creation Guideline](https://nuclei.projectdiscovery.io/templating-guide/)
- [Nuclei Template Matcher Guideline](https://github.com/projectdiscovery/nuclei-templates/wiki/Unique-Template-Matchers)
- [Nuclei Template Contribution Guideline](https://github.com/projectdiscovery/nuclei-templates/blob/master/CONTRIBUTING.md)
- [PD-Community Discord server](https://discord.gg/projectdiscovery)

### CODE REVIEW

**Comment by GeorginaReeder** [comment]
Thanks so much for your contribution @MillerMedia ! :)

**Comment by princechaddha** [comment]
Hello @MillerMedia, The response time to this PR was much longer than usual. Thank you for taking the time to update this template and for contributing to this project 🍻

I’ve made some changes, yet the replace helper still isn’t working as expected. I am checking in with the team to confirm this.

You can also join our [discord server](https://discord.com/invite/projectdiscovery). It's a great place to connect with fellow contributors and stay updated with the latest developments. Thank you once again

File: http/osint/user-enumeration/smugmug.yaml [hunk 1]
```diff
@@ -15,10 +15,13 @@ info:
 
 self-contained: true
 
+variables:
+  valid_username: "{{replace('{{user}}', '.', '')}}"
```
Comment (dwisiswant0) [inline http/osint/user-enumeration/smugmug.yaml:19]:
```suggestion
  valid_username: "{{replace(user, '.', '')}}"
```

File: http/osint/user-enumeration/tumblr.yaml [hunk 2]
```diff
@@ -15,10 +15,13 @@ info:
 
 self-contained: true
 
+variables:
+  valid_username: "{{replace('{{user}}', '.', '')}}"
```
Comment (dwisiswant0) [inline http/osint/user-enumeration/tumblr.yaml:19]:
```suggestion
  valid_username: "{{replace(user, '.', '')}}"
```

**Review by dwisiswant0 — CHANGES_REQUESTED** [review]
when declaring a DSL expr, the functions inside don't need to be wrapped in tags - and it's unnecessary since the var can be used directly.

**Review by dwisiswant0 — APPROVED** [review]

**Review by princechaddha — APPROVED** [review]

**Comment by MillerMedia** [comment]
Thanks for the clarity on this! Glad to see it was approved; thank you!

**Pull request merged**